types/cloudflare-turnstile/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/cloudflare-turnstile",
-    "version": "0.1.9999",
+    "version": "0.2.9999",
```

Comment:
why `.9999`? `0.2.0` sgtm?

Reply:
This is a [requirement](https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library) of the DefinitelyTyped repo

> Note that the version in package.json should only contain major.minor version (e.g. 10.12) followed by .9999

Reply:
TIL